packages/admin-ui-docs/docs/guidelines/components/button.mdx

```diff
@@ -152,3 +181,38 @@ Besides the props listed bellow, you can use all the props accepted by the `butt
 | disabled     | `boolean`                                                                                      | Defines if the Button is disabled         | 🚫       | `false`   |
 | loading      | `boolean`                                                                                      | Defines if the Button is in loading state | 🚫       | `false`   |
 | csx          | `StyleProp`                                                                                    | Defines component styles                  | 🚫       | `{}`      |
+
+</Panel>
+<Panel id="designReference">
+
+## Do’s
+
+1. Buttons should be completely straight-forward. Absolutely no room for ambiguity.
+2. The behavior they trigger must reflect the exact action that is written.
+3. When using icons as buttons, be mindful of accessibility, always include descriptions in the alt tag.
+4. Pay attention to the icon's use in other screens, be consistent of what action it represents.
+
+## Types
+
+### Primary
+
+1. This option should be used for the single most important action of the page.
+   2; The icon must reinforce the importance of the task.
+2. Use this on topbars and confirmation modals.
+3. Use it only once per page / modal.
+
+### Secondary
+
+1. This option should be used for non-urgent, or optional actions.
+2. Use this option for topbars, tables, lists and forms.
```

Comment:
instead of "topbars" it should say "page headers"
table should not be on this list